apps/wekan/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: "3.7"
+
+services:
+  wekan:
+    image: ghcr.io/wekan/wekan:v7.26
+    container_name: wekan
+    environment:
+      # https://github.com/wekan/wekan/blob/main/docker-compose.yml
+      - MONGO_URL=mongodb://wekan-db:27017/wekan
+      - ROOT_URL=https://${APP_DOMAIN}
+      # https://github.com/wekan/wekan/wiki/Troubleshooting-Mail
+      - MAIL_URL=smtp://${MAIL_ADDRESS}:${MAIL_PASSWORD}@smtp.gmail.com:587
+      - MAIL_FROM=Wekan Notifications <noreply.wekan@${APP_DOMAIN}>
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${APP_DATA_DIR}/data/wekan:/data
+    ports:
+      - ${APP_PORT}:8080
+    depends_on:
+      - wekan-db
+    networks:
+      - tipi_main_network
+      - backend
```

Comment:
Please rename to ```wekan_backend_network```

Reply:
https://github.com/runtipi/runtipi-appstore/pull/2051#discussion_r1453758935